tests/protocol/mockwiredata.py

```diff
@@ -160,6 +172,9 @@ def mock_http_get(self, url, *args, **kwargs):
         elif "remoteaccessinfouri" in url:
             content = self.remote_access
             self.call_counts["remoteaccessinfouri"] += 1
+        elif ".vmSettings" in url or ".settings" in url:
```

Comment:
All those url checks are too wide. Eventually I'll marge mockwiredata into mock_wire_protocol and use HttpRequestPredicates instead; right now too many tests depend on mockwiredata.